libdebug/data/thread_list.py

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2023-2024 Gabriele Digregorio. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for details.
+#
+
+
+class ThreadList(list):
+    def __init__(self, debugger, *args):
```

Comment:
Typing information is missing

Reply:
45f93881363a1ae2c28891f3f69bb753110f818b